### ISSUE TITLE
Spots from SAMed for semantic segmentation

### ISSUE THREAD

**Issue by anwai98** [computational-cell-analytics/medico-sam#20]
- ~They finetune for 0 mean and unit variance, instead of SA-1B statistics~
    - (EDIT: I decided to not do this. It defeats the purpose of fineutning and in theory would lead to inferior results)
- [x] Learning rate is 5e-3
- [x] They use a weighted dice-ce loss with weighting to dice at 0.8
- [x] They use AdamW and see that it works better (they use 0.1 as weight decay)
- [x] They use warmup and show in their (a bit shallow) ablation that it leads to good performance.
